ES/ESmy022.xml

```diff
@@ -339,50 +187,202 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                                              <desc>Recent doodles in pencil.</desc>
+                                            </item>
+
+                                            <item xml:id="e3">
+                                               <desc type="findingAid">Notes in the upper margin indicate
+                                                  indicate readings of <ref target="#p2_i1"/> assigned to the
+                                                  days of the week, the hours of the day and the feasts,
+                                                   <locus target="#12rb">fol. 12rb</locus>: <foreign xml:lang="gez">ዘሰሉስ፡</foreign>,
+                                                   <locus target="#15ra">fol. 15ra</locus>: <foreign xml:lang="gez">ዘራቡዕ፡</foreign>
+                                                  <locus target="#18ra">fol. 18ra</locus>: <foreign xml:lang="gez">ዘሐሙስ፡</foreign>,
+                                                  <locus target="#20ra">fol. 20ra</locus>: <foreign xml:lang="gez">ዘዓርብ፡</foreign>,
+                                            <locus target="#22va">fol. 22va</locus>: <foreign xml:lang="gez">ዘቀዳሚት፡</foreign>,
+                                                     <locus target="#26ra">fol. 26ra</locus>: <foreign xml:lang="gez">ዘእኁድ፡</foreign>,
+                                                     <locus target="#32ra">fol. 32ra</locus>: <foreign xml:lang="gez">ዘበዓለ፡ እግዚእ፡</foreign>,
+                                                     <locus target="#33va">fol. 33va</locus>: <foreign xml:lang="gez">ዘጾመ፡፵፡</foreign>,
+                                                     <locus target="#35ra">fol. 35ra</locus>: <foreign xml:lang="gez">ዘጻድቃን፡</foreign>,
+                                                     <locus target="#35vb">fol. 35vb</locus>: <foreign xml:lang="gez">ዘሐዋርያት፡</foreign>,
+                                                    <locus target="#38va">fol. 38va</locus>: <foreign xml:lang="gez">ዘሰማዕት፡</foreign>,
+                                                    <locus target="#41va ">fol. 41va</locus>: <foreign xml:lang="gez">ዘመለእክት፡</foreign>,
+                                                    <locus target="#43rb">fol. 43rb</locus>: <foreign xml:lang="gez">ዘተስአቱ፡ ሰአት፡</foreign>,
+                                                    <locus target="#46rb">fol. 46rb</locus>: <foreign xml:lang="gez">ዘጻድቃን፡</foreign>,
+                                                    <locus target="#50vb">fol. 50vb</locus>: <foreign xml:lang="gez">ዘሰርክ፡</foreign>,
+                                                   </desc>
+                                            </item>
+                                         </list>
+                                      </additions>
+                                      </physDesc>
+                                    </msPart>
+
+                <additional>
```

Comment:
the whole `<additional>` bit must go above, before the first msPart

Reply:
ok